docs/2_2_common_mechanisms.adoc

```diff
@@ -808,6 +808,10 @@ The association between <<clocked-variable,clocked variables>> and their <<Clock
 <<clocked-variable,Clocked variables>> can depend on multiple Clocks.
 _[For example, a global counter could be incremented by multiple tasks, each controlled by a different Clock.]_
 
+Clocks may depend on input clocks.
```

Comment:
Should state here, that "Output Clocks may depend on input Clocks."?

Reply:
@andreas-junghanns since countdown clocks can depend on input clocks not only output but also input clocks can depend on input clocks. (This PR states an example in the xml.) I observed options "Output Clocks and countdown Clocks may depend on input Clocks." and "Clocks ([e.g. output Clocks, countdown Clocks]) may depend on input Clocks". I wasn't happy with neither of them, so I changed it back to "Clocks may depend on input Clocks." fixing "input clocks" to "input Clocks".